README.md

```diff
@@ -86,6 +86,9 @@ $ awesome_bot README.md --allow 403,429
 
 $ awesome_bot README.md --base-url https://github.com/IDR/idr-notebooks/blob/master/
 # will check relative links using the base url provided
+
+$ awesome_bot README.md --base-url https://username:accesstoken@github.com/Organization/Private-repo/blob/master/
```

Comment:
Yeah, wasn't really sure where/how to put it in the README. Do you think it's better to take it out altogether and let people find out organically that it works?